### PR TITLE
Database initial commit

### PR DESCRIPTION
**Pull request by pgonz18**
-Database schema
-Seeding
-Database jest test

### CODE REVIEW

**Comment by pgonz18** [comment]
[Build DB](https://trello.com/c/SyD0h4ue/18-build-db)

File: database/itemModel.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+mongoose.connect('mongodb://localhost:27017/recommended', { useNewUrlParser: true, useUnifiedTopology: true });
```
Comment (jwhunt19) [inline database/itemModel.js:5]: you don't actually need to add the default port, it should default to 27017 automatically.

File: data/items/item.js [hunk 2]
```diff
@@ -0,0 +1,785 @@
...
+  },
+  shipping: {
+    display: faker.helpers.randomize(shipping),
+    eligibility: true,
+  },
+  recommended: {
+    fromShop: [],
+    similarProduct: recommend[i],
+  },
+}));
```
Comment (jwhunt19) [inline data/items/item.js:785]: did you manually input all of the recommended arrays? We need to discuss product ID's

File: test/dbTest.test.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    await connection.close();
+    await mongoose.connection.close();
+  });
+
+  afterEach(async () => {
+    await Item.deleteMany();
+  });
+
+  it('should insert a doc into collection with right shape', async () => {
+    const item = await new Item({
```
Comment (jwhunt19) [inline test/dbTest.test.js:24]: No need to await the creation of an object if it doesn't use anything asynchronously

**Review by jwhunt19 — COMMENTED** [review]
Looks good, have some questions about the seeding before approval

**Comment by jwhunt19** [comment]
Changes made and shown, okay to merge

**Pull request merged**